src/Sylius/Component/Review/Factory/ReviewFactoryInterface.php

```diff
@@ -28,13 +28,13 @@
      *
      * @return ReviewInterface
      */
-    public function createForSubject(ReviewableInterface $subject);
+    public function createForSubject(ReviewableInterface $subject): ReviewInterface;
 
     /**
      * @param ReviewableInterface $subject
      * @param ReviewerInterface|null $reviewer
      *
      * @return ReviewInterface
      */
-    public function createForSubjectWithReviewer(ReviewableInterface $subject, ReviewerInterface $reviewer = null);
+    public function createForSubjectWithReviewer(ReviewableInterface $subject, ?ReviewerInterface $reviewer): ReviewInterface;
```

Comment:
Unfortunately it's required to have it here for guest reviews. I've just made it required to pass the `$reviewer` explicitly as the default value was ridiculous.